xbit/config.js

```diff
@@ -0,0 +1,9 @@
+exports = module.exports = {
```

Comment:
let's not add this file to GIT hub, this is a configuration file. Every body shall copy config.template.js to config.js and modify according to his / her needs